src/external/VTK.jl

```diff
@@ -0,0 +1,59 @@
+import WriteVTK: vtk_grid, vtk_save
+
+# Uses WriteVTK.jl to convert scfres structure to VTk file format
+
+"""
+    save_scfres(filename, scfres, Val(:vtk))
+
+The function takes in the VTK filename and the scfres structure and stores into a VTK file.
+
+Grid Values:
+
+- ρ -> Density in real space
+- ψreal\\_k(i)\\_band(j) -> Real values of Bloch waves in real space
+- ψimag\\_k(i)\\_band(j) -> Imaginary values of Bloch waves in real space
+- ρspin -> Real value of ρspin are stored if ρspin in present
+
+MetaData:
+Energy , EigenValues, Fermi Level and occupation.
+"""
+function save_scfres(filename::AbstractString, scfres::NamedTuple, format::Val{:vts})
+    # Initialzing the VTK Grid
+    basis = scfres.basis
+    grid = zeros(3, basis.fft_size...)
+    for (idcs, r) in zip(CartesianIndices(basis.fft_size), r_vectors_cart(basis))
+        grid[:, Tuple(idcs)...] = r
+    end
+    vtkfile = vtk_grid(filename, grid)
+    
+    # Storing the Bloch Waves in Real space
+    for ik in 1:length(basis.kpoints)
+        for iband in 1:size(scfres.ψ[1])[2]
+            ψ_real = G_to_r(basis, basis.kpoints[ik], scfres.ψ[ik][:,iband])
+            vtkfile["ψ_real_k$(ik)_band$(iband)"] = real.(ψ)
+            vtkfile["ψ_imag_k$(ik)_band$(iband)"] = imag.(ψ)
```

Comment:
Sorry to be asking for another change on this ... names are always tricky. I think actually `ψ_k$(ik)_band$(iband)_real` and  `ψ_k$(ik)_band$(iband)_imag` is actually better, because this makes it more clear that this is the real and the imaginary part of `ψ[ik][:,iband]`. I think that's also what confused you here.

Reply:
Cool! Thanks!